libs/core-styles/src/lib/_imports/elements/form.cms.css

```diff
@@ -0,0 +1,65 @@
+/*
+Forms
+
+Elements that create forms which the user can fill out and submit to the Web site or application.
+
+```
+<form> <fieldset> <legend> <label> <input> <select> <optgroup> <option> <textarea> <button> <output> <progress> <meter> <datalist>
+```
+
+Reference:
+
+- [HTML element reference: Forms | MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Forms)
+
+Styleguide Elements.Forms
+*/
```

Comment:
ℹ️ I migrated this code from CMS, which can now read from it here instead. https://github.com/TACC/Core-CMS/pull/531

Reply:
I migrated this (and other core-styles changes) to https://github.com/TACC/Core-Styles/pull/41/files.